config.js

```diff
@@ -0,0 +1,8 @@
+var config = {};
+
```

Comment:
module.exports = {
  db: '', //type 'mongo'
  url: '' 
};

This is designed for user customization!